jvm/ml4ir-inference/src/main/java/ml4ir/inference/tensorflow/SequenceExampleJavaBuilder.java

```diff
@@ -0,0 +1,58 @@
+package ml4ir.inference.tensorflow;
```

Comment:
the maven build does not compile this file, it is not present in the jar

Reply:
@darshshah do you have a PR which got this code in the jar?  How did you build locally to fix my mixup here?